### ISSUE TITLE
Weird Scrolling Behaviour

### ISSUE THREAD

**Issue by Koomongous** [YTVanced/Vanced#40]
[Video](https://drive.google.com/file/d/1bx_c1Njk8csomlk9p45Wx_OqyWIhEmaO/view?usp=drivesdk)

**Comment by BlackXS** [comment]
Same thing happened only in one video
```
Android 10
Emui 10.1.0
Non-root vanced 15.33.34
Microg 0.2.11.202414
```
![RaFhE6imcdOWU7TPKK](https://user-images.githubusercontent.com/64746796/94330608-fbd56b80-0000-11eb-940e-1cc27b2cf57d.gif)
![xnAxqUZu3amdiMllGv](https://user-images.githubusercontent.com/64746796/94330615-055ed380-0001-11eb-9b7b-a90af275b1b6.gif)
![xXJDQglUxvBhGlW6qc](https://user-images.githubusercontent.com/64746796/94330844-f842e400-0002-11eb-8647-3a495bbc488a.gif)

I even disabled the sponsor block setting with no luck

**Comment by theallen77** [comment]
Having same problem.
Pixel 3a, Android 11

**Comment by BlackXS** [comment]
Fixed with the latest version 15.38.35
Proof:
#
![vit6NBnYA2zpnAM1hH](https://user-images.githubusercontent.com/64746796/94998739-199d6480-05ef-11eb-89f9-1fddc255acfb.gif)

**Comment by VancedOfficial** [comment]
Should be fixed in the 15.38.35 release. Open a new issue if not.

**Comment by theallen77** [comment]
Thanks
